content/administration/maintain/domain_names.rst

```diff
@@ -0,0 +1,315 @@
+============
+Domain names
+============
+
+A **domain name** works as an address for your website. It makes the Internet much more accessible
+as it allows users to type a meaningful web address, such as ``www.odoo.com``, rather than its
+server's IP address with a series of numbers.
+
+You can use a custom domain name to access your Odoo database and websites:
+
+- By :ref:`registering a free domain name with Odoo <domain-name/odoo-register>` (for Odoo Online
+  databases)
+- By :ref:`configuring a custom domain that you already own <domain-name/existing>`.
+
+.. note::
+   Odoo Online and Odoo.sh databases, including their websites, use by default a subdomain of
+   ``odoo.com`` for both the URL and the emails (e.g., ``https://example.odoo.com``).
+
+.. important::
+   Odoo offers a :ref:`free domain name <domain-name/odoo-register>` to all Odoo Online databases
```

Comment:
Feedback: Reading this after the note above, it's not clear to me that the free domain name is not simply a subdomain of odoo.com, which is the very point of this "promotion".

Reply:
Really? ^^'

is this better?

```suggestion
   Odoo offers a :ref:`free custom domain name <domain-name/odoo-register>` of your choice to all
   Odoo Online databases for one year. Visitors can then access your website with an address such as
   ``www.example.com`` rather than the default ``example.odoo.com`````

Reply:
Yes. In the same section, you explain what is a domain and then immediately expect the user to infer that the free domain is not in fact a subdomain of odoo.com, contrary to what is said right above.